documentation/NUnit1032.md

```diff
@@ -14,16 +14,52 @@
 
 An IDisposable field/property should be Disposed in a TearDown method.
 
+This analyzer rule only applies to a `TestFixture` which is using the default
+NUnit `SingleInstance` life cycle where the class is instantiated once for all tests.
+
+If you are using `LifeCycle.InstancePerTestCase` you should dispose the fields/properties
+in the `Dispose` method of the test class.
+
 ## Motivation
 
 Not Disposing fields/properties can cause memory leaks or failing tests.
 
 ## How to fix violations
 
+### LifeCycle.SingleInstance
+
 Dispose any fields/properties that are initialized in `SetUp` or `Test` methods in a `TearDown` method.
 Fields/Properties that are initialized in `OneTimeSetUp`, or with initializers or in constructors
 must be disposed in `OneTimeTearDown`.
 
+### LifeCycle.InstancePerTestCase
+
+If you have `IDisposable` fields or properties, your class must implement the
+[`IDisposable`](https://learn.microsoft.com/en-us/dotnet/api/system.idisposable?view=net-8.0) interface.
+
+Dispose any fields/properties that are initialized at declaration or in the constructor in the
+[`Dispose`](https://learn.microsoft.com/en-us/dotnet/api/system.idisposable.dispose?view=net-8.0) method.
+
+The NUnit.Analyzer will not help you here as the functionality is available in a [Microsoft .NET Analyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.NetAnalyzers)
+
+These are the rules that will help you with this:
```

Comment:
Just some suggestions:
* Removed an "a"
* Added period and merged it into one paragraph.

```suggestion
The NUnit.Analyzer will not help you here as the functionality is available in [Microsoft .NET Analyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.NetAnalyzers). These are the rules that will help you with this:
```